workalendar/america/barbados.py

```diff
@@ -45,16 +43,16 @@ class property. All holidays are shifted to Monday if on a Sunday
         return days
 
     def get_variable_days(self, year):
-        ch_days = super(Barbados, self).get_variable_days(year)
+        christian_days = super(Barbados, self).get_variable_days(year)
```

Comment:
In my latest commit I've changed this variable name to `western_days` as it contains New year's eve.